### PR TITLE
Fix broken call of sourceCode

### PR DESCRIPTION
**Pull request by ADimeo**
This fixes a broken (on my system) method call.

The method CompiledMethod sourceCode does not exist on my system. CompiledMethod getSource does, however. I do not know how the former call ended up in our image, since none of the SWE-images include this call.

### CODE REVIEW

**Comment by arminWel** [comment]
The method `#getSource:` is not implemented(at least not in my image), while `#sourceCode:` is. Therefore is must recline this PR.  Reasons for the differences in the images should be discussed with the chair as soon as possible(maybe monday morning? @ADimeo).

**Comment by arminWel** [comment]
For some reason (those must be quite weird), my Image know has the method the way you have. We should definitely ask at the chair about that-

**Review by arminWel — APPROVED** [review]

**Comment by arminWel** [comment]
There are problems with older versions of squeak. Thus have to be solved, before we merge. I'm working on it.

File: packages/CollabCodeEdit-Tests.package/CCEModelTest.class/instance/tearDown.st [hunk 1]
```diff
@@ -1,4 +1,5 @@
 running
 tearDown
 	model changed: #close.
-	self newMorphs do: [:morph | morph model changed: #close]
\ No newline at end of file
+	self newMorphs do: [:morph | morph model changed: #close].
```
Comment (PaulBrachmann) [inline packages/CollabCodeEdit-Tests.package/CCEModelTest.class/instance/tearDown.st]:
```suggestion
	self newMorphs do: [ :morph | morph model changed: #close ].
```

File: packages/CollabCodeEdit-Tests.package/CCEPackageManagerTests.class/instance/testHashchangesAfterMetaClassInformationChange.st [hunk 2]
```diff
@@ -2,18 +2,13 @@ tests
...
-	CCESession default scope: #( 'Signals' ).
-	
-	testClass := (PackageInfo named: 'Signals') classes first.
-	
+	CCESession default scope: #( 'CollabCodeEdit-Tests' ).
+	testClass := Environment default classNamed: 'CCETestDummy'.
 	originalComment := testClass comment.
- 
-	hashBeforeChange := packageManager hashPackages: #( 'Signals' ).
+	[ hashBeforeChange := packageManager hashPackages: #( 'CollabCodeEdit-Tests' ).
```
Comment (PaulBrachmann) [inline packages/CollabCodeEdit-Tests.package/CCEPackageManagerTests.class/instance/testHashchangesAfterMetaClassInformationChange.st:8]: Code following a multiline block being opened should be indented. (See lines below.)

File: packages/CollabCodeEdit-Tests.package/CCEPackageManagerTests.class/instance/testHashchangesAfterMetaClassInformationChange.st [hunk 3]
```diff
@@ -2,18 +2,13 @@ tests
...
+	hashAfterChange := packageManager hashPackages: #( 'CollabCodeEdit-Tests' ).
 	
- 	hashAfterChange := packageManager hashPackages: #( 'Signals' ).
-	
-	self deny: hashBeforeChange equals: hashAfterChange.
-	
-	testClass comment: originalComment
\ No newline at end of file
+	self deny: hashBeforeChange = hashAfterChange. ]
+	ensure: [testClass comment: originalComment]
```
Comment (PaulBrachmann) [inline packages/CollabCodeEdit-Tests.package/CCEPackageManagerTests.class/instance/testHashchangesAfterMetaClassInformationChange.st]:
```suggestion
	ensure: [ testClass comment: originalComment ]
```

Comment (PaulBrachmann) [inline packages/CollabCodeEdit-Tests.package/CCEPackageManagerTests.class/instance/testHashchangesAfterMetaClassInformationChange.st]: The current indentation makes it unclear that `ensure:` belongs to the preceding block.

File: packages/CollabCodeEdit-Tests.package/CCEPackageManagerTests.class/instance/testHashchangesAfterMetaClassInformationChange.st [hunk 4]
```diff
@@ -2,18 +2,13 @@ tests
...
 	testClass comment: 'This comment was changed by packageManagerTest'.
+	hashAfterChange := packageManager hashPackages: #( 'CollabCodeEdit-Tests' ).
 	
- 	hashAfterChange := packageManager hashPackages: #( 'Signals' ).
-	
-	self deny: hashBeforeChange equals: hashAfterChange.
-	
-	testClass comment: originalComment
\ No newline at end of file
+	self deny: hashBeforeChange = hashAfterChange. ]
```
Comment (PaulBrachmann) [inline packages/CollabCodeEdit-Tests.package/CCEPackageManagerTests.class/instance/testHashchangesAfterMetaClassInformationChange.st]:
```suggestion
	self deny: hashBeforeChange = hashAfterChange ]
```

**Review by PaulBrachmann — CHANGES_REQUESTED** [review]
Looks great. It is indeed quite the relief to finally see a build passing!

There are however some minor style inconsistencies that should be fixed.
For further information, refer to our [Smalltalk Style Guide](https://docs.google.com/document/d/11PfOTymUnovjca5ExAI6-6-cC16f6q5mQ8juo9Wn13I/edit?usp=sharing).

**Pull request merged**